tests/nativeapp/test_run_processor.py

```diff
@@ -794,6 +794,7 @@ def test_create_dev_app_create_new_quoted(
                 - src: app/streamlit/*.py
                 dest: ui/
 
+
```

Comment:
Is this `dest:ui/` placed here intentionally? It's an extra value, that is not used in schema? According to design doc, we should not accept extra value, so this causes an error. Should it be deleted?

Reply:
@sfc-gh-bgoel @sfc-gh-cgorrie can you advise?

Reply:
I think this is an indentation error -- this is part of the last entry of the artifacts collection (i.e. `{ src: "app/streamlit/*.py", dest: "ui/" }`).